src/main/java/uk/ac/ebi/eva/utils/FileUtils.java

```diff
@@ -66,10 +65,13 @@ public static File getResource(String resourcePath) {
         return new File(FileUtils.class.getResource(resourcePath).getFile());
     }
 
-    public static Properties getPropertiesFile(String propertiesFilePath) throws IOException {
-        InputStream input = new FileInputStream(propertiesFilePath);
+    public static InputStream getResourceAsStream(String resourcePath) {
+        return FileUtils.class.getResourceAsStream(resourcePath);
```

Comment:
`class` shouldn't be necessary, see usage without it in `VariantVcfExacFactory.java`.